PDSim/core/core.py

```diff
@@ -1243,7 +1286,9 @@ def OBJECTIVE_CYCLE(self, Td_Tlumps0, X, epsilon = 0.003, cycle_integrator = 'RK
             print(error_ascii_bar(error_metric, epsilon), 'cycle-cycle    ', error_metric)
             print(error_ascii_bar(abs(mdot_error), 1), 'mdot [%]', mdot_error, '|| in:', mdot_in*1000, 'g/s || out:', mdot_out*1000, 'g/s ')
             
-            worst_error = max(abs(self.lumps_resid[0]), abs(self.resid_Td), np.sqrt(np.sum(np.power(errors, 2))))
+            worst_error = max(np.max(np.abs(self.lumps_resid)), 
```

Comment:
What is up with this change?  It seems to change nothing because the np.max is operating on a single value?

Reply:
**self.lumps_resid** is either a single array value for a single-lump, or an array or residuals. By using np.max and np.abs, we select always the worst residual among the lumps or lump. I tested it with both single-lump and multi-lump.

Reply:
Sorry, got it, that makes sense, I got lost in all the ( )